File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int l=0;
        int r=nums.size()-1;
        while(l<=r){
            if(nums[l]==val){
                nums[l] = nums[r];
                r--;
                continue;
            }
            l++;
        }
        return l;
    }
};

File: Data Structures & Algorithms/longest-increasing-subsequence/submission-1.py
from bisect import bisect_left
class Solution:
    def lengthOfLIS(self, nums: List[int]) -> int:
        res=[]
        for n in nums:
            i = bisect_left(res,n)
            if i == len(res):
                res.append(n)
            else:
                res[i]=n
        return len(res)